### PR TITLE
Create organizations screen for user portal

### PR DESCRIPTION
**Pull request by beingnoble03**
**What kind of change does this PR introduce?**
GSoC follow up PR --> Creating Organizations screen for user portal

**Snapshots/Videos:**

image

**Did you add tests for your changes?**
No. It's the target for next week.

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**
No.

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by beingnoble03** [comment]
No existing tests are failing. The code coverage tests are failing due to new components. Writing tests are these components in the target for the next week.

**Comment by beingnoble03** [comment]
@evasharma12 I've added the required tests which was the target for this week. Please review. So, that we can merge this branch into develop, once this PR gets merged into admin-user-portal. Thanks.

**Review by evasharma12 — APPROVED** [review]
Looks good to me.

**Pull request merged**
